tests/cli.rs

```diff
@@ -338,18 +433,19 @@ fn test_depool_body() -> Result<(), Box<dyn std::error::Error>> {
 #[test]
 fn test_depool_1() -> Result<(), Box<dyn std::error::Error>> {
     let depool_abi = "tests/samples/fakeDepool.abi.json";
-    let depool_addr = "0:cf72b41b704b7173467ffcd2c7bbc2a30d251996e3e3d848a74f9f72c8bc65e6";
-    let msig_addr = "0:507fc74745d5a259b9939dfbdfd97cd186d13e8a7160206f3054746c1f0518cd";
     let seed_phrase = "blanket time net universe ketchup maid way poem scatter blur limit drill";
 
     let mut cmd = Command::cargo_bin(BIN_NAME)?;
-    cmd.arg("config")
-        .arg("--wallet")
-        .arg(msig_addr)
-        .arg("--addr")
-        .arg(depool_addr)
-        .assert()
-        .success();
+    let out = cmd.arg("config")
+        .arg("--list")
+        .output()
+        .expect("Failed to get config.");
+
+    let out = String::from_utf8_lossy(&out.stdout).to_string();
+
+    let mut depool_addr = out;
+    depool_addr.replace_range(..depool_addr.find("addr").unwrap_or(0) + 8, "");
+    depool_addr.replace_range(depool_addr.find('"').unwrap_or(depool_addr.len()).., "");    
```

Comment:
Move this block to a separate function to refactor it easily in one place if we will change config command or config parameters in future. And then call this function in every test if needed.